force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,18 @@
+trigger AccountTrigger on Account (before insert, after insert) {
+    
+    if (Trigger.IsBefore && Trigger.IsInsert) {
+        // Set the account type to prospect
+        AccountTriggerHelper.setAccountTypeProspect(Trigger.new);
+        
+        // Copy the shipping address to the billing address.
+        AccountTriggerHelper.setAccountMailAddrToBillAddr(Trigger.new);
+
+    }
```

Comment:
Food for thought is there a way to loop account once for the trigger handler calls? This will make your code more efficient because it reduces iteration time. This is not something you do all the time but good to try out.